cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackend.java

```diff
@@ -17,27 +17,37 @@
  */
 package com.telefonica.iot.cygnus.backends.orion;
 
+import java.io.UnsupportedEncodingException;
+
+import org.json.JSONException;
+
 import com.telefonica.iot.cygnus.backends.http.JsonResponse;
+import com.telefonica.iot.cygnus.errors.CygnusBadAuthorization;
+import com.telefonica.iot.cygnus.errors.CygnusPersistenceError;
+import com.telefonica.iot.cygnus.errors.CygnusRuntimeError;
 
 /**
  *
  * @author pcoello25
  */
 public interface OrionBackend {
-    
+
     /**
-     * Updates a context element given a list of statistical metadata for each entity's attribute.
+     * Updates a context element given a list of statistical metadata for each
+     * entity's attribute.
      * 
      * @param entityId
      * @param entityType
      * @param allAttrStats
      * @throws Exception
      */
     // TBD: https://github.com/telefonicaid/fiware-cygnus/issues/304
-    // void updateContext(String entityId, String entityType, ArrayList<OrionStats> allAttrStats) throws Exception;
-    
+    // void updateContext(String entityId, String entityType,
+    // ArrayList<OrionStats> allAttrStats) throws Exception;
+
     /**
-     * Subscribe to Orion with a given port, host and string with subscription information (v1).
+     * Subscribe to Orion with a given port, host and string with subscription
+     * information (v1).
```

Comment:
v1 -> NGSIv1

Reply:
We'll make some changes. But tests fail because properties files don't have original credentials, we think that we push credentials, it's a bad idea. Any idea?

Reply:
Yes, pushing real credential is a bad idea given the repository is public. Maybe you could mock the auth componenents and use fake credentials with them... but not sure how simple/complex would be.

(Anycase, to be completely clear, in this particular comment I was referreing to just change "v1" by "NGSIv1" in the JavaDoc comment).